content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+<div class="alert alert-info">
+Datadog Profiling is in beta. Reach out to <a href="/help/">Datadog Support</a> if you encounter any issues or have feedback to share.
+</div>
+
+## Setup
+
+Profiling libraries are shipped within the following tracing language library. Select your language below to learn how to enable profiling for your application:
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling library is supported OpenJDK 11, Oracle Java 8 with a commercial license, Oracle Java 11, and Zulu Java 8. All JVM-based languages, i.e. Scala, Groovy, Kotlin, Clojure, etc. are supported. To begin profiling applications:
```

Comment:
Turns out I was wrong about it working with Oracle 8 with commercial license, sorry!